maps/vgstation.dmm

```diff
@@ -0,0 +1 @@
+step_x = 0
```

Comment:
yes I am testing travis
this is awful travis doesn't work the way I think it does